### DISCUSSION TITLE
Contoh BOT autorespon sederhana dengan PHP

### DISCUSSION THREAD

**Discussion by maschai** [kirimwa-id/forum#145] [Umum]
Mas rio / para master 

boleh diberikan contoh bot autorespon sederhana dengan PHP seperti tutorial BOT dengan nodejs .....?

saya haturkan terimakasih sebesar-besarnya atas bantuannya

**Comment by rioastamal** [comment]
@maschai 

1. Pastikan [webhook](https://developer.kirimwa.id/api/v1/#webhooks) udah disetup. 
2. Pada script webhook cek isi pesan dari pengirim dan [reply](https://developer.kirimwa.id/api/v1/#post-messages) jika format pesan sesuai yang diinginkan.
